src/Symfony/tests/Integration/OtelSdkBundle/DependencyInjection/config/exporters/config.yaml

```diff
@@ -7,24 +7,17 @@ trace:
   exporters:
     # dsn example
     exporter1: zipkin+http://zipkinhost:1234/path
-    # dsn key example
-    exporter2:
-      dsn: jaeger+https://foo:bar@jaegerhost:1234/path/to/endpoint?name=foo
     # parameter example
-    exporter3:
+    exporter2:
       type: newrelic
```

Comment:
change this to zipkin? (newrelic exporter no longer a thing)

Reply:
I opted to switch this to otlp (seeing as there were no tests for it before)